### DISCUSSION TITLE
Undefined reference to RdKafka::Error::create

### DISCUSSION THREAD

**Discussion by steviekm3** [edenhill/librdkafka#4644] [General]
Am using 1.8.2 version and this is only undefined reference to a function that I see.  I did nm -C on both librdkafka.a and librdkafka++.a files and cannot see this function.

Anyone know of fix for this ?  I suspect maybe this is option when building.

I am creating Error object so I can check if the error is_retriable.

**Comment by mfarrugi** [comment]
There appears to be no definition for `RdKafka::Error::create`. There's one for `RdKafka::ErrorImpl::create`, but that's not part of the public API.
